index.html

```diff
@@ -183,6 +270,32 @@ <h1 class="project-heading">Multi-Post<br>Stories</h1>
                     </span>
                 </div>
             </div>
+            <div class="details-container-desktop">
+                <div class="heading-and-highlights">
+                    <h1 class="project-heading">Uber Navigation</h1>
+                    <ul class="highlights">
+                        <li class="project-type">Uber</li>
+                        <li><img src="./Images/Counter.png" alt="counter" class="counter1"></li>
+                        <li class="which-end">Lead Developer</li>
+                        <li><img src="./Images/Counter.png" alt="counter" class="counter2"></li>
+                        <li class="project-year">2018</li>
+                    </ul>
+                </div>
+                <div class="project-description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </div>
+                <ul class="tags">
+                    <li class="html">html</li>
+                    <li class="ruby-on-rails">ruby on rails</li>
+                    <li class="css">css</li>
+                    <li class="javascript">javascript</li>
+                </ul>
+                <div class="project-button">
+                    <span class="project-text">
+                        See Project
+                    </span>
+                </div>
+            </div>
```

Comment:
- [ ] You did an amazing job creating all your cards :+1:, but I have noticed that the last one is overlapping with the About me section. Kindly make your card and the next section separate to improve UX.

**Your design**

![Screenshot from 2023-01-26 06-36-40](https://user-images.githubusercontent.com/105117832/214828690-b57af930-ad34-4c97-94e6-795d6f250886.png)

Reply:
I have tried everything to fix it but it doesn't fix. Could you please tell me how to fix it?

Reply:
Hi @sameedmohsin28 did you try changing the `position: relative;` on your CSS? Sometimes it causes some problems as well as using `position: absolute;`

Reply:
I think it would be better to use flexbox because it makes it easy to build responsive projects :+1:, but if you don't want to do that I have noticed that you can fix that by giving a max-height to each project. For example for the third project you could write:

```
.third-project {
  ...
  max-height: 600px;
}
```